src/tests/unit/tests/injected/visualization/drawer.test.ts

```diff
@@ -2,6 +2,7 @@
 // Licensed under the MIT License.
 import { IMock, It, Mock, Times } from 'typemoq';
 import { IActionN } from 'typemoq/_all';
+import { isFunction } from 'lodash';
```

Comment:
Build failing with ERROR: /Users/vsts/agent/2.149.2/work/1/s/src/tests/unit/tests/injected/visualization/drawer.test.ts:5:1 - Import sources within a group must be alphabetized.